components/sensor/nextion.rst

```diff
@@ -0,0 +1,176 @@
+.. _nextion_sensor:
+
+Nextion Sensor Component
+========================
+
+.. seo::
+    :description: Instructions for setting up Nextion sensor.
+    :image: nextion.jpg    
+
+The ``nextion`` sensor platform supports intergers or floats (Xfloat). It can be a component, a variable or a waveform in the Nextion display.
+It is best to set the components vscope to global in the Nextion Editor. This way the component will be available
+if the page is shown or not. 
+
+.. note::
+
+    The Nextion can receive an interger but it can only send 3 bytes for a negative integer. The range if using the :ref:`nextion_custom_sensor_protocol` is:
```

Comment:
```suggestion
    The Nextion can receive an integer but it can only send 3 bytes for a negative integer. The range if using the :ref:`nextion_custom_sensor_protocol` is:
```